src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/DefaultProjectConfigurationFilePathStore.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Composition;
+using Microsoft.CodeAnalysis.Razor;
+using Microsoft.CodeAnalysis.Razor.ProjectSystem;
+
+namespace Microsoft.VisualStudio.LanguageServerClient.Razor
+{
+    [Shared]
+    [Export(typeof(ProjectConfigurationFilePathStore))]
+    internal class DefaultProjectConfigurationFilePathStore : ProjectConfigurationFilePathStore
+    {
+        private readonly Dictionary<string, string> _mappings;
+        private readonly object _mappingsLock;
+
+        public override event EventHandler<ProjectConfigurationFilePathChangedEventArgs> Changed;
```

Comment:
Just confirming keeping events above the constructor (instead of with properties below the constructor) is intentional?

Reply:
Yup, the event is a field